libcontainer/rootfs_linux.go

```diff
@@ -38,7 +38,7 @@ type mountConfig struct {
 	cgroup2Path     string
 	rootlessCgroups bool
 	cgroupns        bool
-	file            *os.File
+	fileFd          *int
```

Comment:
perhaps just `fd` as it's a tautology otherwise (file file descriptor).

Reply:
Right. Changed that and simplified variables names (mountFileFdsList --> mountFds, _LIBCONTAINER_MOUNT_FILE_FDS --> _LIBCONTAINER_MOUNT_FDS, mountFile* --> MountFd*)